docs/project/changelog.md

```diff
@@ -14,6 +14,24 @@ myst:
 
 # Change Log
 
+## Version 0.27.3
+
+_Insert Date Here_
+
+- Added the `context` parameter to `WebLoop.create_task()` {pr}`5431`
+- {{ Fix }} `mountNativeFS` API now correctly propagates the error. {pr}`5434`
+- {{ Fix }} Since 0.27.1, Pyodide has been broken in iOS because iOS ships
+  broken wasm-gc support. Pyodide feature detects whether the runtime supports
+  wasm-gc and uses it if it is present. Unfortunately, iOS passes the feature
+  detection but wasm-gc doesn't work as expected. {pr}`5445`
+
+### Packages
+
+- Upgraded `narwhals` to 1.24.1 {pr}`5386`
+- Upgraded Pyxel to 2.3.6 {pr}`5427`
```

Comment:
```suggestion
```

I think this one should be removed as #5427 only updated the checksum for the project, and the upgrade is being tried out in #5435.

Reply:
Thanks for the explanation, I misunderstood.